handler/api_stub.go

```diff
@@ -0,0 +1,116 @@
+package handler
+
+import (
+	"encoding/json"
+	"errors"
+	"io/ioutil"
+	"net/http"
+
+	"github.com/ONSdigital/go-ns/log"
+	"gopkg.in/yaml.v2"
+)
+
+var (
+	authHeaderKey     = "Authorization"
+	userAuthHeaderKey = "X-Florence-Token"
+
+	internalSeverError = Response{Message: "internal server error", Status: 500}
+	badRequestError    = Response{Message: "bad request", Status: 400}
+	unauthorizedError  = Response{Message: "not authenticated", Status: 401}
+)
+
+type APIStub struct {
+	scenarios Scenarios
+}
+
+func NewAPIStub() (*APIStub, error) {
+	source, err := ioutil.ReadFile("config.yml")
+	if err != nil {
+		log.ErrorC("error attempting to load config.yml", err, nil)
+		return nil, err
+	}
+
+	var scenarios Scenarios
+	if err := yaml.Unmarshal(source, &scenarios); err != nil {
+		log.ErrorC("error attempting to marshal config.yml", err, nil)
+		return nil, err
+	}
+
+	b, err := json.MarshalIndent(scenarios, "", " ")
+	if err != nil {
+		return nil, errors.New("failed to marshal config to json")
```

Comment:
Can the error be logged here

Reply:
Yes it can......but am i going to do it!.........................okay i've decided to do it